src/Client/Update/SpreadsheetUpdate.fs

```diff
@@ -152,12 +166,16 @@ module Spreadsheet =
                 nextState, model, Cmd.none
             | UpdateArcFile arcFile ->
                 let reset = state.ActiveView.ArcFileHasView(arcFile) //verify that active view is still valid
-
+               
                 let nextState =
-                    if reset then
-                        Spreadsheet.Model.init (arcFile)
-                    else
-                        { state with ArcFile = Some arcFile }
+                    let baseState =
+                        if reset then
+                            Spreadsheet.Model.init (arcFile)
+                        else
+                            { state with ArcFile = Some arcFile }
+                    match arcFile with
+                    | ArcFiles.DataMap _ -> { baseState with ActiveView = ActiveView.DataMap }
```

Comment:
Propably makes sense to remove ActiveView.DataMap completly. Handle the single datamap table like i handled it for templates